microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1306,358 +1306,243 @@
-            .style("fill", "#747678")
-            .style("font-size", "14px")
-            .style("line-height", "24px")
-            .style("font-family", "Mulish")
-            .style("font-weight", "400")
-            .style("font-style", "normal")
-            .text("(population with a");
-
-        svg.append("text")
-            .attr("x", 110)
-            .attr("y", -radius + 245)
-            .style("fill", "#747678")
-            .style("font-size", "14px")
-            .style("line-height", "24px")
-            .style("font-family", "Mulish")
-            .style("font-weight", "400")
-            .style("font-style", "normal")
-            .text("hunter-gatherer lifestyle)");
-
-        svg.append("text")
-            .attr("x", -radius + 125)
-            .attr("y", 35)
-            .style("fill", "#FC8900")
-            .style("font-size", "72px")
-            .style("font-family", "Mulish")
-            .style("font-weight", "700")
-            .style("line-height", "normal")
-            .style("text-anchor", "middle")
-            .text(diversityInSampleValue);
+    if (diversityInSampleValue == lowestDiversityScore) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.90, radius * 0.59, 22);
     }
 
+    if (diversityInSampleValue == highestDiversityScore) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.97, radius * 0.46, -46);
+    }
+
+    if (diversityInSampleValue > lowestDiversityScore && diversityInSampleValue <= 2.99) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.75, radius * 0.45, 13);
+    }
+
+    if (diversityInSampleValue >= 3.0 && diversityInSampleValue <= 3.99) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.65, radius * 0.30, 3);
+    }
+
+    if (diversityInSampleValue >= 4.0 && diversityInSampleValue <= 4.99) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.30, radius * 0.35, 8);
+    }
+
+    if (diversityInSampleValue >= 5.0 && diversityInSampleValue <= 5.99) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.10, radius * 0.35, -30);
+    }
+
+    if (diversityInSampleValue >= 6.0 && diversityInSampleValue <= 6.49) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.25, radius * 0.55, 5);
+    }
+
+    if (diversityInSampleValue >= 6.5 && diversityInSampleValue <= 6.99) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.25, radius * 0.45, 5);
+    }
+
+    if (diversityInSampleValue >= 7.0 && diversityInSampleValue < highestDiversityScore) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.45, radius * 0.25, -35);
+    }
```

Comment:
Now that the line for the diversity score is being drawn via d3 rather than trying to position a nested SVG, can you please condense all of these conditional statements? We know what the angle is, so shouldn't it just be a matter of drawing a line from the desired inner radius to the desired outer radius? If that's not the case, please explain the issue that requires these conditionals.

Reply:
I'll send an email with explanation and screenshots.